Portugues.json

```
@@ -0,0 +1,159 @@
+            "closed_window": "O aplicativo Goggle Chrome foi fechado , portanto o login não foi efetuado com sucesso ."
+        },
+        "error_code": "Login error code: {error_code}",
+        "unexpected_content": "Algum erro inexperado aconteceu, geralmente por rediricionamento de internet. Voce precissa logar em algum site de internet para ter acesso?",
+        "incorrect_login_pass": "Senha ou usuario incorreto",
+        "incorrect_email_code": "Codigo de email incorreto",
+        "incorrect_twofa_code": "Codigo F2A incorreto",
+        "email_code_required": "Codigo no email requerido. Checar sua caixa de entrada",
+        "twofa_code_required": "Token F2A Requerido."
+    },
+    "error": {
+        "captcha": "Seu login foi negado pelo CAPTCHA.\nPor favor tente em aproximadamente +12 houras.",
+        "site_down": "Twitch esta fora do ar, tentando em {seconds} segundos...",
+        "no_connection": "Sem conecao com a TWITCH, reconectando em {seconds} segundos..."
+    },
+    "gui": {
+        "output": "Saida",
+        "status": {
+            "name": "Status",
+            "idle": "Em espera",
+            "exiting": "Saindo",
+            "terminated": "Finalizado",
+            "watching": "Assistindo o {channel}",
+            "cleanup": "Limpando os Canais...",
+            "gathering": "Coletando Canais...",
+            "switching": "Mudando o canal...",
+            "fetching_inventory": "Coletando Inventario...",
+            "fetching_campaigns": "Coletando campanhas...",
+            "adding_campaigns": "Adicionando as campanhas para o sistema... {counter}"
+        },
+        "tabs": {
+            "main": "Tela Principal",
+            "inventory": "Inventario",
+            "settings": "Configuracoes",
+            "help": "Ajuda"
+        },
+        "tray": {
+            "notification_title": "Mined Drop",
+            "minimize": "Minimize to Tray",
+            "show": "Mostrar",
+            "quit": "Sair"
+        },
+        "login": {
+            "name": "Tela de login",
+            "labels": "Status:\nUser ID:",
+            "logged_in": "Entrou",
+            "logged_out": "Saiu",
+            "logging_in": "Penetrando...",
+            "required": "Login Requerido",
+            "request": "Por favor, entre para continuar.",
+            "username": "Usuario",
+            "password": "Senha",
+            "twofa_code": "Codigo 2FA (optional)",
+            "button": "Entrar"
+        },
+        "websocket": {
+            "name": "Status Websocket",
+            "websocket": "Websocket #{id}:",
+            "initializing": "Inizializando...",
+            "connected": "Conectado",
+            "disconnected": "Disconectafo",
+            "connecting": "Conectando...",
+            "disconnecting": "Disconectando...",
+            "reconnecting": "Reconectando..."
+        },
+        "progress": {
+            "name": "Progresso da campanha",
+            "drop": "Drop:",

```

Small addition per https://github.com/DevilXD/TwitchDropsMiner/commit/83ffea013b5d7c80f5e51fa5373b30e35fea9ece

```suggestion
            "drop": "Drop:",
            "game": "Jogo:",
```